### DISCUSSION TITLE
Why there are less katas than solved?

### DISCUSSION THREAD

**Discussion by MaxBlack-dev** [codewars/codewars.com#2592] [General]
There are 9209 katas on Codewars, but user Voile(https://www.codewars.com/users/Voile) solved 9747!
How come user solved more katas than exits?
This is unfair if you can't beat someone!

**Comment by hobovsky** [comment]
Voile solved kata which are not there.

No, seriously now: 

This specific user has been on Codewars for a very long time, and he is said to solve almost every kata which is, or was, ever published. Including kata which later got deleted, unpublished, or retired.

Maybe Voile could share with us how many kata he still have not solved? I remember i asked once, but I am ~~not sure if I recall correctly the figure of  300?~~ sure i dont remember the answer ;)

**Reply by MaxBlack-dev** [reply]
> Voile solved kata which are not there.
> 
> No, seriously now:
> 
> This specific user has been on Codewars for a very long time, and he is said to solve almost every kata which is, or was, ever published. Including kata which later got deleted, unpublished, or retired.
> 
> Maybe Voile could share with us how many kata he still have not solved? I remember i asked once, but I am ~not sure if I recall correctly the figure of 300?~ sure i dont remember the answer ;)

If there are katas that was deleted they should not be a part of Leaderboards! Because it's unfair if one user have advantages, Codewars should be fair! Otherwise what is the point of competing!

**Reply by JohanWiltink** [reply]
How are you entitled to a level playing field with a competitor who has been around longer than you?

Having been here longer is definitely an advantage, but making it not an advantage is impossible, and calling that unfair is not useful ( and does not make you sound good either ).

Or would you judge _everybody_ by their very last solve _only?_

**Reply by hobovsky** [reply]
> If there are katas that was deleted they should not be a part of Leaderboards! Because it's unfair if one user have advantages, Codewars should be fair! Otherwise what is the point of competing!

I have no strong opinion on whether retired and unpublished kata should be counted, but I have a few thoughts:

- Competition is not the only way Codewars can be used. Many users use it for learning, and they do not really care how many kata other users solved. It's possible to just learn together, and to not compete.
- Counting retired and unpublished kata towards leaderboards position can be considered fair by some, because it rewards users for their participation in beta process and rewards them for actively maintaining the content of Codewars kata library.
- There's just no point competing with some users (not only Voile). Can't touch this. I would be even ready to challenge you to try ;)

**Reply by hobovsky** [reply]
Additionally, amount of completed kata is not a competitive stat on Codewars. There's a leaderboard which uses honor points awarded for solved kata, and in case of retired betas, Voile (or anyone else) was not awarded with too many honor points. A retired beta grants like 6 points in total, and no rank reward. You can somewhat make up for this difference with other kata.

**Comment by DonaldKellett** [comment]
Kata are only retired, not deleted. I assume that Voile's completed Kata
statistics include retired ones. Anyway, as long as Voile didn't cheat and
there's no bug in the completed Kata calculation logic, I don't see how
this is unfair?

**Reply by MaxBlack-dev** [reply]
No one can't get same amount of score as Voile, and you think this is fair? I believe it's truly unfair, because everyone should have same chance to get same score!

**Reply by JohanWiltink** [reply]
The world isn't fair. Keep yelling. Unsubscribe.

**Reply by MaxBlack-dev** [reply]
> The world isn't fair. Keep yelling. Unsubscribe.

Great solution, very professional!

**Reply by lechevalier** [reply]
> No one can't get same amount of score as Voile, and you think this is fair? I believe it's truly unfair, because everyone should have same chance to get same score!

Seriously no.
Voile and other users that are members of CW since many years have solved more kata, even kata that have been retired.  
Points were given to them as a reward for the time they spent on those tasks.
Subtracting these points is totally unfair because they did work more than you or any other recent user.
Moreover it removes the incentive to work on new kata for CW.

Not everyone should have same chance to get same score: CW is not a time limited competition.
Its leaderboard is global: it translates not only the amount of work you put in kata solving, but also the amount of time you spent improving the website.

If you want a chance to get the same score as someone else, enter a time-limited competition.
There are many resources on internet where you can find such competition.
Otherwise you should stop complaining when it is only fair to reward user loyalty and seniority.
